module/input/SpeechIntent/data/config/SpeechIntent.yaml

```diff
@@ -0,0 +1,3 @@
+# Controls the minimum log level that NUClear log will display
+log_level: DEBUG
+debug_mode: true
```

Comment:
I feel like "debug_mode" should just be using the nuclear log level debug

Reply:
It's been removed in favour or a command line argument "file" for testing.